src/main.ts

```diff
@@ -45,3 +45,9 @@ if (require.main === module) {
 }
 
 export default run
+
+// Force embed of version in build files from package.json for release check
+/* eslint-disable eslint-comments/no-unlimited-disable */
+/* eslint-disable */
+require('../package.json').name
+require('../package.json').version
```

Comment:
This will be a problem if we want to move to ESM, and is not ideal even now.

We could abuse the `ncc` `--license` flag during the compilation to embed the version as well. I'll leave you the decision to explore this alternative solution now, or we could do it later when it's really an issue.

Reply:
Had the same idea but sadly the generated doesn't include versions, and I believe that the fact that `datadog-synthetics-github-action` has been added there at all is due to the `package.json` import.

Only proper alternative I see for now is having a explicit build+package script that writes the build version somewhere manually.

Reply:
Oh, I see, `--license <file>` won't take the license from `<file>`, but it'll write the licenses to `<file>`.

I agree, the only alternative I see is to copy the version as part of the `package` script.